jax/experimental/jax2tf/jax2tf.py

```diff
@@ -3174,7 +3180,8 @@ def _pjit(*args: TfVal,
   sharded_args: Sequence[TfVal] = tuple(
       map(_shard_value, args, _in_avals, in_shardings))
   results = _interpret_jaxpr(jaxpr, *sharded_args,
-                             extra_name_stack=util.wrap_name(name, "pjit"))
+                             extra_name_stack=util.wrap_name(name,
+                                                             "jit" if config.jax_jit_pjit_api_merge else "pjit"))
```

Comment:
I reverted to just saying "pjit" rather than have a conditional. But I am tempted to just say "jit", because soon there will be no pjit. Actually, is the plan that we keep the pjit_p primitive?

Reply:
Yes, jit and pjit will continue to exist. I can explain more offline but there will still be 2 APIs as per the current plan.